diff --git a/Makefile b/Makefile
index 220a7e5..be81881 100644
--- a/Makefile
+++ b/Makefile
@@ -1,8 +1,3 @@
-# Make sure we have 'git' and it works OK:
-ifeq ($(shell which git),)
-    $(error 'git' is not installed on this system)
-endif
-GITVER ?= $(word 3,$(shell git --version))
 
 NAME = git-hub
 LIB = lib
@@ -17,11 +12,11 @@ EXTS = $(shell find $(EXT) -type f) \
 SHARE = share
 
 # XXX Make these vars look like git.git/Makefile style
-PREFIX ?= /usr/local
+PREFIX ?= $(out)
 # XXX Using sed for now. Would like to use bash or make syntax.
 # If GIT_EXEC_PATH is set, `git --exec-path` will contain that appended to the
 # front. We just want the path where git is actually installed:
-INSTALL_LIB ?= $(shell git --exec-path | sed 's/.*://')
+INSTALL_LIB ?= $(PREFIX)/bin
 INSTALL_CMD ?= $(INSTALL_LIB)/$(NAME)
 INSTALL_EXT ?= $(INSTALL_LIB)/$(NAME).d
 INSTALL_MAN1 ?= $(PREFIX)/share/man/man1
